cmd/ddev/cmd/dotddev_assets/commands/host/sequel.example

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+## #ddev-generated
+## Description: Run Sequel Ace or SequelPro against current db
+## Usage: sequel
+## Example: "ddev sequel"
+
+# This command uses $DDEV_HOST_DB_PORT to get the port for the connection
+#
+# Both apps are free, but available only for macOS-only 
+# Find Sequel Ace in the Mac App Store or at https://sequel-ace.com/
+# Find Sequel Pro at https://www.sequelpro.com/
+
+query="mysql://root:root@127.0.0.1:${DDEV_HOST_DB_PORT}/db"
```

Comment:
This query strategy unfortunately doesn't work with Sequel Pro, only with Sequel Ace. 
<img width="421" alt="Sequel_Pro" src="https://user-images.githubusercontent.com/112444/89957545-eb099b80-dbf4-11ea-9d30-72ed502b3ba6.png">

Reply:
Not sure if you've seen this, but if Sequel Pro ever gets up off the mat (and [maybe it will](https://github.com/sequelpro/sequelpro/issues/3705#issuecomment-653959299)), it will support these schemes. The [nightly builds](https://sequelpro.com/test-builds) already do (of course the most recent of these is over a year old, whatever that says).

That doesn't solve anything for people using the current release on the sequelpro website, of course.

If I ruled the world, I might be inclined to deprecate support for Sequel Pro with the hope that its actual released version becomes less crashy.

I could also see if it's possible to create that spf file without writing it to disk and pipe it into an `open` command.  Seems like a bit of a hassle, but ddev has support sequelpro for long enough (I'm guessing) that removing it would be uncharitable.

Reply:
I do hate that these similar tools have completely different launch options. 

I think maybe we should add sequel-ace as an official shell command (like launch), based on this one, and also provide an *example* for sequel-pro.  And remove the existing golang one. 

Then our problem is how to hide it if Sequel-Ace isn't installed or if we're not on macOS. That requires a bit of extra effort. I guess I'll work on https://github.com/drud/ddev/issues/2209 and try to make it so individual commands can determine whether they show up?